### DISCUSSION TITLE
User agent header to fix issue after new privacy policy twitter update

### DISCUSSION THREAD

**Discussion by bo-tadpol** [himanshudabas/twint#4] [General]
Hi @himanshudabas - I started using your version of twint about 2 months ago. Specifically the **twint-fixes** branch. After Twitter's recent change to show a privacy policy page listing supported browsers, you need to make the following change on line 51 in **token.py**.

```twitter_request = Request('GET', self.url, headers={'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:78.0) Gecko/20100101 Firefox/78.0'}).prepare()```

Also, if you are open to collaborating moving forward - I would love to have write access and then I can open PRs for you to review.

**Comment by himanshudabas** [comment]
Sure, I'll provide you with write access.

Although I am a bit busy these days and haven't been able to contribute much, but I'd be coming back with ome fixes in a couple weeks.
